serialzy/serializers/base_model.py

```diff
@@ -68,6 +71,8 @@ def deserialize_from_dir(source: BinaryIO, read_from_dir) -> Any:
 
 
 class ModelBaseSerializer(DefaultSchemaSerializerByReference, ABC):
+    META_FILE_NAME = 'meta.json'
```

Comment:
I'm not a fan of CAPS at class-level constants.
Maybe in the future or in descendants this field would be a property.